### PR TITLE
Fullscreen fix

### PR DESCRIPTION
**Pull request by sekwah41**
this should fix your fullscreen issues @roldanjr

### CODE REVIEW

File: app/renderer/public/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: 'unsafe-inline'">
```
Comment (sekwah41) [inline app/renderer/public/index.html:7]: Just gets rid of a warning and allows us to be a bit more secure/specific with loaded scripts

File: app/main/src/main.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ function createMainWindow() {
 	win = new BrowserWindow({
 		width: 340,
 		height: getFrameHeight(),
-		resizable: true,
+		resizable: false,
```
Comment (sekwah41) [inline app/main/src/main.ts:76]: If its only being set via code this shouldn't be true ever. It seems altering between true and false makes mac confused for full sizing.

**Review by roldanjr — APPROVED** [review]
LGTM! Thank you @sekwah41

**Comment by roldanjr** [comment]
I'll try to test it again on Mac once the build has been done :)

**Comment by sekwah41** [comment]
Ill look into it a bit more after work as on mac it seems to vanish sometimes though that may be in dev and on windows it seems to fullscreen but not show fullscreen or something weird. So ill try to figure out what is wrong with it and the resizing option being set or unset.

**Comment by sekwah41** [comment]
@roldanjr after these last changes it seems to be working fine on mac now

File: app/main/src/main.ts (hunk 2 above)
Comment (13banda) [inline app/main/src/main.ts:76]: in ubuntu it break  fullscreen so after set it true it work again
so please check it on ubuntu

Comment (sekwah41) [reply app/main/src/main.ts:76]: Weirdly setting this to true breaks it on mac. Or at least i believe it was this property. So I have a task this weekend to look into it with all 3 systems open at once to find one that doesn't break something :(

**Pull request merged**